nilearn/_utils/__init__.py

```diff
@@ -200,8 +200,8 @@ def all_classes(
 
 
 __all__ = [
-    "_repr_niimgs",
     "CacheMixin",
+    "_repr_niimgs",
```

Comment:
unrelated: probably worth renaming this function to non private if we expose it as part of the public part of the `_utils` subpackage

Reply:
ha thanks!
Sorry I did not necessarily mean to change it in this PR, but I'll take it in any case.